lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -459,6 +460,20 @@ def non_fractional_currency?(currency)
       def emv_payment?(payment)
         payment.respond_to?(:emv?) && payment.emv?
       end
+
+      def error_code_from(response)
```

Comment:
This can be simplified as: 

```ruby
code = response['error']['code'] 
decline_code = response['error']['decline_code'] 

if decline_code && code == 'card_declined'
  STANDARD_ERROR_CODE_MAPPING[decline_code] 
else
  STANDARD_ERROR_CODE_MAPPING[code] 
end
```